app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+## class User
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'user_id'
+  has_many :comments, foreign_key: 'user_id'
+  has_many :likes, foreign_key: 'user_id'
```

Comment:
- [ ] Good job on this so far, but if we reference the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png) you would notice that the user foreign key is referenced with author. So please kindly change the `user_id` to `author_id`